hs_file_types/utils.py

```diff
@@ -243,3 +255,20 @@ def set_logical_file_type(res, user, file_id, hs_file_type=None, folder_path='',
     except:
         if fail_feedback:
             raise
+
+
+def get_model_program_aggregations(user):
```

Comment:
This method should be able to use filters too, and will enable using a filter in my previous comment